src/controllers/usersController.js

```diff
@@ -133,25 +135,37 @@ const loginUser = async (req, res, next) => {
   if (success) {
     const { email, password } = req.body;
 
-    await verifyLoginCredentials(req.app.get('db'), email, password)
+    await getHashedPasswordFromEmail(req.app.get('db'), email)
```

Comment:
Can we combine verifyLoginCredentials() and getHashedPasswordFromEmail() methods into one and rename it to something like getLoginCredentials()? Cause for one the verifyLoginCredentials() method doesn't really do any verification. Secondly, the two methods make a call to the same table with the same query, just to get different fields. It would look cleaner if it's only one method, and you get both UserID and Password from the User table at once.

Reply:
Yeah good point, I'll make that change!